### ISSUE TITLE
Console Fire crit kill crashes

### ISSUE THREAD

**Issue by RockoTDF** [Sandrem/FlyCasual#1576]
I saw a previous report of this that had been closed, but it's still happening.  I've attached two screenshots to show.  The point where the ruler goes down is where you don't get any attack options, and never do for the rest of the game.  I think the problem may be that console fire is supposed to let you shoot before you die (or at least ought to since the ship isn't "destroyed" right away and should still be able to attack), the ship is removed, and then there's no shooter left to shoot.

<img width="1680" alt="screen shot 2019-02-14 at 22 43 17" src="https://user-images.githubusercontent.com/47582934/52833628-abb7c880-30ab-11e9-9733-10cc1817f2a2.png">
<img width="1680" alt="screen shot 2019-02-14 at 22 46 49" src="https://user-images.githubusercontent.com/47582934/52833629-abb7c880-30ab-11e9-931b-c0925422d128.png">

**Comment by rune-b** [comment]
I got the same error. When a ship is destroyed from the Console Fire die roll, it is not allowed to attack back (which it should according to the simultaneous attack rule) and the error appears when clicking on the next ship to activate

**Comment by SpiderMana16** [comment]
![screen shot 2019-02-24 at 2 35 34 pm](https://user-images.githubusercontent.com/47953163/53305419-0a690900-3847-11e9-8da7-785d24db568e.png)

Same here. The ship destroyed by Console Fire disappeared immediately, and the app couldn't figure out what to do as it was his turn to attack, but he... wasn't there. Next game I played, I killed Iden at i4, so she got to shoot due to simultaneous fire rulings, and was appropriately removed at the end of that initiative step.
